instrumentation-api/src/test/java/io/opentelemetry/instrumentation/api/internal/SupportabilityMetricsTest.java

```diff
@@ -66,14 +73,17 @@ void resetsCountsEachReport() {
             Config.create(Collections.singletonMap("otel.javaagent.debug", "true")), reports::add);
 
     metrics.recordSuppressedSpan(SpanKind.CLIENT, "favoriteInstrumentation");
+    metrics.incrementCounter("some counter");
 
     metrics.report();
     metrics.report();
 
     assertThat(reports)
         .isNotEmpty()
-        .hasSize(1)
+        .hasSize(2)
```

Comment:
Just noticed now so no big deal, but we generally wouldn't call `isNotEmpty` just before a `hasSize` check